back/lib/tasks/single_use/20240613_rename_gv_profile.rake

```diff
@@ -0,0 +1,30 @@
+namespace :gv_transition do
+  desc 'Replace the Citizenlab default moderator with Go Vocal attributes.'
+  task :rename_profile, [] => [:environment] do
+    reporter = ScriptReporter.new
+    Tenant.safe_switch_each do |tenant|
+      default_moderators = User.where(email: %w[moderator@citizenlab.co moderator@govocal.com])
+      default_moderators.each do |moderator|
+        if moderator.first_name.casecmp('citizenlab').zero?
```

Comment:
Rubocop told me to "Use moderator.first_name.casecmp('citizenlab').zero? instead of moderator.first_name.downcase == 'citizenlab'.". I find it very readable so I added a [CDM point](https://www.notion.so/citizenlab/Back-end-Code-Discussion-Meeting-BE-CDM-a94ef3cfdd9849e4bde22d2b10dcffeb?pvs=4#9eeb8998267b4d7fb292f53cc9bd5bd2).

Reply:
Yeah, I agree - certainly in this instance `downcase` is much more readable